test/src/module/storage/azureTest.c

```diff
@@ -288,9 +288,9 @@ testRun(void)
                 hrnCfgArgRawZ(argList, cfgOptRepoType, STORAGE_AZURE_TYPE);
                 hrnCfgArgRawZ(argList, cfgOptRepoPath, "/");
                 hrnCfgArgRawZ(argList, cfgOptRepoAzureContainer, TEST_CONTAINER);
-                hrnCfgArgRaw(argList, cfgOptRepoAzureHost, hrnServerHost());
-                hrnCfgArgRawFmt(argList, cfgOptRepoAzurePort, "%u", hrnServerPort(0));
-                hrnCfgArgRawBool(argList, cfgOptRepoAzureVerifyTls, testContainer());
+                hrnCfgArgRaw(argList, cfgOptRepoStorageHost, hrnServerHost());
+                hrnCfgArgRawFmt(argList, cfgOptRepoStoragePort, "%u", hrnServerPort(0));
+                hrnCfgArgRawBool(argList, cfgOptRepoStorageVerifyTls, testContainer());
```

Comment:
We don't seem to test the ca-file/ca-path stuff for Azure anywhere - is there a reason for that?

Reply:
We don't test it in S3 either, because I don't have a good idea how to do so. The values get passed directly to OpenSSL which does not allow you to inspect them (as far as I can see).

We sort of test that they work in ioTlsTest.c by passing bogus values and expecting an error. But if you pass one bogus value it works fine so it's hard to know if both are set correctly. I suppose we could propagate that to the storage tests, but it seems like more work that it is worth.

Another thought is to store them in TlsClient and output them in logging. That won't prove that they are getting set but it will at least prove that they made it to the TlsClient.

Whatever we do it will be a fair amount of work and I'd rather not tie it to this commit. Ideas?

Reply:
May put this comment on the GCS review and come up with a general solution since GCS does not test this either?

Reply:
Maybe we should create a project card in the pgbackrest board. In general, I think a user would know pretty quickly if there was some problem in this area. The bigger problem for us is testing these things, which I think is probably a project all on its own....

Reply:
Card added: https://github.com/pgbackrest/pgbackrest/projects/2#card-56037030